drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/DecisionTableAnalyzerConflictResolverTest.java

```diff
@@ -0,0 +1,162 @@
+import org.drools.workbench.models.guided.dtable.shared.model.GuidedDecisionTable52;
+import org.drools.workbench.screens.guided.dtable.client.resources.i18n.AnalysisConstants;
+import org.drools.workbench.screens.guided.dtable.client.widget.analysis.AnalyzerProvider;
+import org.drools.workbench.screens.guided.dtable.client.widget.analysis.DataBuilderProvider;
+import org.drools.workbench.screens.guided.dtable.client.widget.analysis.DecisionTableAnalyzer;
+import org.drools.workbench.screens.guided.dtable.client.widget.analysis.ValidateEvent;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.drools.workbench.screens.guided.dtable.client.widget.analysis.TestUtil.*;
+import static org.junit.Assert.*;
+
+@RunWith( GwtMockitoTestRunner.class )
+public class DecisionTableAnalyzerConflictResolverTest {
+
+    @GwtMock
+    AnalysisConstants analysisConstants;
+
+    @GwtMock
+    DateTimeFormat dateTimeFormat;
+
+    private AnalyzerProvider analyzerProvider;
+
+    @Before
+    public void setUp() throws Exception {
+        analyzerProvider = new AnalyzerProvider();
+    }
+
+    @Test
+    public void testNoIssue() throws Exception {
+        final DecisionTableAnalyzer analyzer = analyzerProvider.makeAnalyser()
+                                                               .withPersonAgeColumn( ">" )
+                                                               .withAccountDepositColumn( "<" )
+                                                               .withApplicationApprovedSetField()
+                                                               .withData( DataBuilderProvider
+                                                                                  .row( 100, 0.0, true )
+                                                                                  .end() )
+                                                               .buildAnalyzer();
+
+        analyzer.onValidate( new ValidateEvent( Collections.emptyList() ) );
+        assertDoesNotContain( "ConflictingRows", analyzerProvider.getAnalysisReport() );
+    }
+
+    @Test
+    public void testNoIssueWithNulls() throws Exception {
+        final GuidedDecisionTable52 table52 = analyzerProvider.makeAnalyser()
+                                                              .withPersonAgeColumn( ">" )
+                                                              .withPersonAgeColumn( "<" )
+                                                              .withData( DataBuilderProvider
+                                                                                 .row( null, null )
+                                                                                 .end() )
+                                                              .buildTable();
+
+
+        // After a save has been done, the server side sometimes sets the String field value to "" for numbers, even when the data type is a number
```

Comment:
Yes, caused by [this](https://github.com/droolsjbpm/drools/commit/bd2b052e425688e214574048763bce1b9e3cf6dc#diff-c1898016135de0a31337480ccd605a3cR45). so when a ```DTCellValue52``` is deserialized and the default constructor called the value is set to a ```String```